src/server.js

```diff
@@ -296,6 +296,31 @@ function initializeClient(socket, client, generateToken, token) {
 		}
 	});
 
+	socket.on("push:register", (subscription) => {
+		if (!client.isRegistered() || !client.config.sessions[token]) {
+			return;
+		}
+
+		const registration = client.registerPushSubscription(client.config.sessions[token], subscription);
+
+		if (registration) {
+			client.manager.webPush.pushSingle(client, registration, {
+				type: "notification",
+				timestamp: Date.now(),
+				title: "The Lounge",
+				body: "🚀 Push notifications have been enabled"
+			});
```

Comment:
Web push is all new stuff, so I'd rather have a reassuring confirmation that it actually worked.